app-frontend/src/app/pages/projects/detail/exports/exports.controller.js

```diff
@@ -0,0 +1,56 @@
+export default class ProjectDetailExportsController {
+    constructor($state, $scope, $timeout, projectService) {
+        'ngInject';
+        this.$state = $state;
+        this.$scope = $scope;
+        this.$timeout = $timeout;
+        this.projectService = projectService;
+    }
+
+    $onInit() {
+        this.isLoadingProject = true;
+        this.$scope.$parent.$ctrl.fetchProject().then(p => {
+            this.project = p;
+            this.isLoadingProject = false;
+            this.populateExportList(this.$state.params.page || 1);
+        });
+    }
+
+    populateExportList(page) {
+        if (this.isLoadingExports) {
+            return;
+        }
+        delete this.errorMsg;
+        this.isLoadingExports = true;
+        // save off selected scenes so you don't lose them during the refresh
+        this.exportList = [];
+        this.projectService.listExports(
+            {
+                sort: 'createdAt,desc',
+                pageSize: '10',
+                page: page - 1,
+                project: this.project.id
+            }
+        ).then(exportResult => {
+            this.lastExportResult = exportResult;
+            this.numPaginationButtons = Math.max(6 - exportResult.page % 10, 3);
```

Comment:
Looks like this is a variation on something we've done is a bunch of places but it doesn't look like we actually ever use the value...so that's interesting.